decidim-meetings/app/views/decidim/meetings/meetings/_meeting.html.erb

```diff
@@ -0,0 +1,144 @@
+<%= render layout: "layouts/decidim/shared/drawer", locals: { close_url: meetings_path } do %>
+  <div class="meeting__container">
+
+    <h1 class="h2 decorator" data-heading-tag><%= present(meeting).title(links: true, html_escape: true ) %></h1>
```

Comment:
Also, please note that e.g. the agenda items heading level is currently incorrect but it would be correct after changing this to `<h2>`.

While we are waiting for a better solution for the drawers in all, all drawers should be treated as modals that open on top of the page. And when you think about that further isn't it pretty weird we have a separate URL for every meeting and the first title you see on that page is `<h2>`?

Reply:
Same as previous, we'll back into this once drawer issues have been solved.